src/code/fv-components/location-indicator.js

```diff
@@ -0,0 +1,18 @@
+import React, {PropTypes} from 'react';
```

Comment:
Is it worth making this its own component? Sam made a comment earlier that one of my component's was just text in a div and probably not worth breaking out, so I'm trying to figure out best practices.